signac_dashboard/modules/statepoint_list.py

```diff
@@ -7,18 +7,37 @@
 
 
 class StatepointList(Module):
-    """Displays the job state point."""
+    """Displays the job state point.
+
+    :param exclude_const: Only show statepoint parameters that change values
+        if True (default: :code:`False`)
+    :type exclude_const: bool
+    """
+
     def __init__(self,
                  name='Statepoint Parameters',
                  context='JobContext',
                  template='cards/statepoint_list.html',
+                 exclude_const=False,
                  **kwargs):
         super().__init__(name=name,
                          context=context,
                          template=template,
                          **kwargs)
+        self.exclude_const = exclude_const
 
     def get_cards(self, job):
-        sp = OrderedDict(sorted(job.statepoint().items(), key=lambda t: t[0]))
+        sp = [(k, v) for k, v in sorted(job.statepoint().items(),
+              key=lambda t: t[0]) if k in self.sp_keys_to_show]
+        sp = OrderedDict(sp)
```

Comment:
I don't think this will work for state points with nested dicts. The schema will have dotted keys. 🌵
```suggestion
        sp = [(k, v) for k, v in sorted(job.statepoint().items(),
              key=lambda t: t[0]) if k in self.statepoint_keys_to_show]
        sp = OrderedDict(sp)
```